pine64/rock64/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  boot = {
+    initrd.kernelModules = [
+      "pcie_rockchip_host"
+      "phy_rockchip_pcie"
+      "sdhci_pci"
```

Comment:
Is this not detected by nixos-generate-config?

Reply:
Mhm. but I guess our installer wouldn't boot, so the normal install workflow wouldn't boot.

Reply:
https://github.com/NixOS/nixos-hardware/blob/master/pine64/rockpro64/default.nix#L3


my local config (and, in turn, what was done in this PR) is inspired by how the RockPro64 is configured

Reply:
I'm currently using a half-functional build of u-boot/tow-boot, generated from this PR https://github.com/Tow-Boot/Tow-Boot/pull/322

Reply:
We can leave those modules in here if you also add a README how to bootstrap the rest of the configuration i.e. instructions to build a installer image.

Reply:
it's just the generic aarch64 uefi installer iso

I guess it'd be more common to go the u-boot+sd-card image, like it says on the wiki

which is unlike what I have -- tow-boot+normal EFI installer

I wouldn't really recommend people use a draft PR for their own stuff though